backend/src/routes/index.js

```diff
@@ -186,7 +186,7 @@ router.patch('/events/:eventId/questions/:questionId', ensureLogin, rateLimiter(
 
   if (action == 'like' && !req.user && computerId.startsWith('nobiri-')) return next({ status: 401 })
 
-  if (!['like', 'unlike', 'pin', 'unpin'].includes(action)) return next({ status: 400 })
+  if (!['like', 'unlike', 'pin', 'unpin', 'archive', 'unarchive'].includes(action)) return next({ status: 400 })
```

Comment:
we need to put these in an allowedActions array

Reply:
done